apps/aeprimop/src/aeprimop.erl

```diff
@@ -2120,19 +2307,20 @@ assert_contract_call_stack(CallStack, S) ->
         _Other -> runtime_error(nonempty_call_stack)
     end.
 
+-spec assert_auth_call_object_not_exist(aect_call:call(), state()) -> ok | no_return().
 assert_auth_call_object_not_exist(Call, S) ->
     AuthCallId = aect_call:id(Call),
     Pubkey     = aect_call:caller_pubkey(Call),
     case find_auth_call(Pubkey, AuthCallId, S) of
         none -> ok;
-        {value, _} -> runtime_error(auth_call_object_already_exist)
+        {_, _} -> runtime_error(auth_call_object_already_exist)
```

Comment:
Maybe just `_`?

Reply:
'{_, _}' is consistent through functions here. Are you sure '_' pattern won't break things? Yes, I tested and it seems fine, but may be there is some obscure case in vivo?